src/web/components/Card/components/MediaMeta.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { css } from 'emotion';
+
+import { palette } from '@guardian/src-foundations';
+import { textSans } from '@guardian/src-foundations/typography';
+
+import Audio from '@frontend/static/icons/audio.svg';
+import Photo from '@frontend/static/icons/photo.svg';
+import Video from '@frontend/static/icons/video.svg';
+
+type Props = {
+    mediaType: MediaType;
+    pillar: Pillar;
+    mediaDuration?: number;
+};
+
+const iconWrapperStyles = ({ pillar, mediaType }: Props) => css`
+    width: 1.5rem;
+    height: 1.4375rem;
+    background-color: ${palette[pillar].main};
+    border-radius: 50%;
+    display: inline-block;
+
+    > svg {
+        width: 0.875rem;
+        height: auto;
+        margin-left: auto;
+        margin-right: auto;
+        margin-top: 0.375rem;
+        display: block;
+        transform: ${mediaType === 'Video' ? `translateY(0.0625rem)` : ``};
+    }
+`;
+
+const durationStyles = ({ pillar }: Props) => css`
+    color: ${palette[pillar].main};
+    font-size: 0.75rem;
+    line-height: 1rem;
+    ${textSans.xsmall()}
+    font-weight: 600;
+`;
+
+const wrapperStyles = css`
+    display: flex;
+    align-items: center;
+`;
+
+export function secondsToDuration(secs?: number): string {
+    if (typeof secs === `undefined` || secs === 0) {
+        return ``;
+    }
+    const dtime = new Date(secs * 1000).toISOString();
+    const duration = secs > 3600 ? dtime.substr(11, 8) : dtime.substr(14, 5);
+    return /^0/.test(duration) ? duration.substr(1) : duration;
+}
```

Comment:
This is longer but removes the need for string manipulation:
```typescript
export function secondsToDuration(secs?: number): string {
    if (typeof secs === `undefined` || secs === 0) {
        return ``;
    }

    const seconds = Number(secs);
    var h = Math.floor(seconds / 3600);
    var m = Math.floor((seconds % 3600) / 60);
    var s = Math.floor((seconds % 3600) % 60);

    let duration = [];
    if (h > 0) {
        duration.push(h);
    }
    if (m > 0 || h === 0) {
        // supports 0:59
        duration.push(m);
    }
    if (s > 0) {
        duration.push(s);
    }
    return duration.join(':');
}
```
What do you think?

Reply:
I've a sweet spot for one-liners and concise functions, but your version is much more idiomatic and semantically sound. Changed.